scripts/entrypoint.sh

```diff
@@ -26,13 +26,22 @@ set -eu
 # turn on bash's job control
 set -m
 
+log_in_json() 
+{
+    FILENAME="${0##*/}"
```

Comment:
Hadn't seen `"${0##*/}"` before 😄

Reply:
_From stack overflow_ - "for the variable $0, and the pattern '/', the two hashes mean from the beginning of the parameter, delete the longest (or greedy) match—up to and including the pattern. So, where $0 is the name of a file, eg., $HOME/documents/doc.txt, then the parameter would be expanded as: doc.txt" :D